addons/web/static/src/core/model_selector/model_selector.scss

```diff
@@ -7,4 +7,22 @@
             width: inherit;
         }
     }
+
+    .o_dropdown_button {
+        float: right;
+        margin-top: -20px;
```

Comment:
not sure about these two rules. That looks a bit weak. I couldn't find something similar for other autocomplete dropdowns. Why do you need it?